src/Router/Router.ts

```diff
@@ -138,7 +145,14 @@ export default class Router {
   }
 
   async executed(txRequest: any): Promise<TxStatus> {
-    await this.actions.cleanup(txRequest);
+    /**
+     * We don't cleanup because cleanup needs refactor according to latest logic in EAC
```

Comment:
We need another transition to check for these conditions.

Reply:
Sure we do, but I would leave it to other PR since it won't be common situation probably that timenode will find its own transactions to cancel. I believe timenodes in 80% cases won't be owners of scheduled txs.

Reply:
Okay